cmd/kubectl-image/import.go

```diff
@@ -30,7 +31,9 @@ func init() {
 	imageimport.Flags().StringP("namespace", "n", "", "namespace to use")
 	imageimport.Flags().StringP("source", "s", "", "image source for the import")
 	imageimport.Flags().Bool("mirror", false, "mirror the image")
-	imageimport.Flags().Bool("insecure-source", false, "skip tls check for the remote registry")
+	imageimport.Flags().Bool("no-mirror", false, "do not mirror the image")
+	imageimport.Flags().Bool("insecure-source", false, "skip remote registry tls")
+	imageimport.Flags().Bool("no-insecure-source", false, "do not skip remote registry tls")
```

Comment:
Initially I thought it was possible to negate boolean fields (as in `--no-bool`), but [the feature is not yet in place](https://github.com/spf13/pflag/issues/214). Sorry for the confusion on this subject.

Inspired on [this upstream example](https://github.com/knative/client/pull/346), I think we should also hide the flags `--no-mirror` and `--no-insecure-source` as well.

Reply:
We can do that, no problem. May I ask you what is the reasoning behind hiding them ?

Reply:
OK, I have hidden them. Can we have this in ?